sdk/resourcemanager/azure-resourcemanager-appservice/src/test/java/com/azure/resourcemanager/appservice/FunctionAppsTests.java

```diff
@@ -584,6 +588,8 @@ public void canCreateAndUpdateFunctionAppOnAcaWithPrivateRegistryImage() {
 
         functionApp.update()
             .withMaxReplicas(15)
+            .withPrivateRegistryImage("xiaofeiacr.azurecr.io/samples/nginx:latest", "https://xiaofeiacr.azurecr.io")
+            .withCredentials("xiaofeiacr", "PASSWORD")
```

Comment:
Should we call `TestBase.getPassword()`?

Reply:
The test is disabled due to private registry image is needed. Guess we can keep it as is?

Reply:
OK